astroid/nodes/as_string.py

```diff
@@ -441,6 +442,10 @@ def visit_pass(self, node) -> str:
         """return an astroid.Pass node as string"""
         return "pass"
 
+    def visit_partialfunction(self, node: objects.PartialFunction) -> str:
+        """Return an objects.PartialFunction as string."""
+        return self.visit_functiondef(node)
```

Comment:
This is better than crashing, but it's not a great representation as it's identical to the regular function. Oh well.